dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+    'MK': [Macedônia do NORTH, Macedónia do NORTH, República da Macedônia do NORTH, República da Macedónia do NORTH, Macedóni*, Escópia, Skopje, Skopie]
+    'MT': [Malta, República de Malta, Maltês*, Valeta]
+    'PT': [Portugal, República Portuguesa, Português*, Portugues*, Lisboa]
+    'RS': [Sérvia, República da Sérvia*, Sérvi*, Belgrado]
+    'SI': [Eslovénia, Eslovênia, República da Eslovénia, Esloven*, Liubliana, Ljubljana]
+    'SM': [San Marino, Sereníssima República de San Marino, República de San Marino, São Marinho, São Marino, Samarinês*, São-Marinense*, São-Marinhense*, San-Marinense*]
+    'VA': [Vaticano, Cidade do Vaticano, Estado da Cidade do Vaticano, Vatican*]
+
+  WEST:
+    'AT': [Áustria, República da Áustria, Austríac*, Viena]
+    'BE': [Bélgica, Reino da Bélgica, Belga*, Bélgic*, Bruxelas, Região da Capital Bruxelas]
+    'CH': [Suíça, Confederação Suíça, Suíç*, Zurique, Berna]
+    'DE': [Alemanha, República Federal da Alemanha, Alemã*, Berlim, Frankfurt am Main, Frankfurt sobre o Meno, Francoforte do Meno, Frankfurt, Francoforte]
+    'FR': [França, República Francesa, Francês*, Frances*, Paris]
+    'LI': [Liechtenstein, Listenstaine, Principado de Liechtenstein, Principado do Liechtenstein, Liechtensteinense*, Liechtensteiniense*, Liechtensteinian*, Listenstainian*, Listenstainês*, Vaduz]
+    'LU': [Luxemburgo, Grão-Ducado do Luxemburgo, Grão-Ducado de Luxemburgo, Luxemburguês*, Cidade do Luxemburgo]
+    'MC': [Mónaco, Mônaco, Principado do Mónaco, Principado de Mônaco, Monegasc*]
+    'NL': [Países Baixos, Holanda, Neerlandês*, Holandês*, Amesterdão, Amsterdão, Amsterdã, Amesterdã]
+
+OCEANIA:
+  AU-NZ:
+    'AU': [Austrália, Comunidade da Austrália, Australian*, Aussie*, Oz, Camberra, Sydney]
+    'CK': [Ilhas Cook, Cookian*, Cookense*, Avarua]
+    'NF': [Ilha Norfolk, Ilha Norfolque, Norfolquin*]
+    'NZ': [Nova Zelândia, Neozelandês*, Kiwi*, Wellington, Auckland, Auclanda]
+
+  MEL:
+    'FJ': [Fiji, República das Fiji, República das Fíji, República de Fiji, República de Fíji, Fidji, Fijian*]
+    'NC': [Nova Caledónia, Nova Caledônia, Neocaledóni*, Neocaledôni*, Numeá, Numea, Numeia]
+    'PG': [Papua-Nova Guiné, Papua Nova Guiné, Estado Independente da Papua-Nova Guiné, Papuási*, Porto Moresby, Port Moresby]
+    'SB': [Ilhas Salomão, Salomónic*, Salomônic*, Salomonense*, Honiara]
+    'VU': [Vanuatu, República de Vanuatu, Vanuatuense*, Porto Vila]
+
+  MIC:
+    'FM': [Estados Federados da Micronésia, EFM, Micronésia, Micronési*, Paliquir]
+    'GU': [Guam, Guão, Guame, Guamês*, Guames*, Aganha]
+    'KI': [Kiribati, Quiribáti, Quiribati, República de Kiribati, Kiribatian*, Etemaquian*, Kiribatin*, Taraua, Taraua do SOUTH, Tarawa do SOUTH]
+    'MH': [Ilhas Marshall, República das Ilhas Marshall, Marshallin*, Marshalês*, Majuro]
+    'MP': [Ilhas Marianas Setentrionais,  Ilhas Marianas do NORTH, Comunidade das Ilhas Marianas Setentrionais, NORTH-Marianense*, Capital Hill, Saipã, Saipan]
```

Comment:
Do you have to capitalize "NORTH"?

Reply:
Sorry, this was a mistake in adjusting the final file. I will fix it.